pom.xml

```diff
@@ -80,6 +89,13 @@
 		</dependency>
 
 		<dependency>
```

Comment:
I know it is not written somewhere, but one of the design decisions of the library is to be as dependency light as posible. Why do we need guava?